services/aur/aur.service.js

```diff
@@ -168,8 +174,85 @@ class AurVersion extends BaseAurService {
   }
 }
 
+class AurMaintainer extends BaseAurService {
+  static get category() {
+    return 'social'
+  }
+
+  static get route() {
+    return {
+      base: 'aur/maintainer',
+      pattern: ':packageName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'AUR maintainer',
+        namedParams: { packageName: 'google-chrome' },
+        staticPreview: this.render({ maintainer: 'luzifer' }),
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'maintainer' }
+  }
+
+  static render({ maintainer }) {
+    return { message: maintainer, color: 'blue' }
+  }
+
+  async handle({ packageName }) {
+    const json = await this.fetch({ packageName })
+    return this.constructor.render({ maintainer: json.results.Maintainer })
```

Comment:
Given that we had to update the schema to account for the `Maintainer` key having a null value, we'll need to check for that scenario and handle it accordingly. Something like the below could work (obligatory disclaimer about potential prettier/eslint/etc. issues with inline code typed in the GitHub UI), note you'd need to also add the import `InvalidResponse` from `..` up on line 7

```suggestion
    const { results: { Maintainer: maintainer } } = await this.fetch({ packageName })
    if (!maintainer) {
      throw new InvalidResponse({ prettyMessage: 'No maintainer' })
    }
    return this.constructor.render({ maintainer })
```

Reply:
Thanks for the great insight and example! I've missed that fail-safe. although, i thought there's some sort of generic error handle that catch them.